improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -393,93 +394,13 @@ def test_error(self):
-        )
-        condition = "<"
-        prob_threshold = 0.5
-        expected = (
-            "cubes.extract(Constraint(name="
-            "'probability_of_rainfall_rate_above_threshold',"
-            " coord_values={'threshold': 0.03})"
-            ")[0].data < 0.5"
-        )
-        result = plugin.construct_condition(constraint_value, condition, prob_threshold)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, expected)
-
-    def test_works_with_lists(self):
-        """Test that the construct_condition method works with a list
-        of Constraints. """
-        plugin = WeatherSymbols()
-        constraint_list = [
-            str(
-                iris.Constraint(
-                    name="probability_of_lwe_snowfall_rate_above_threshold",
-                    coord_values={"threshold": 0.03},
-                )
-            ),
-            "-",
-            str(
-                iris.Constraint(
-                    name="probability_of_rainfall_rate_above_threshold",
-                    coord_values={"threshold": 0.03},
-                )
-            ),
-            "*",
-            "0.7",
-        ]
-        condition = "<"
-        prob_threshold = 0.5
-        expected = (
-            "( cubes.extract(Constraint(name="
-            "'probability_of_lwe_snowfall_rate_above_threshold', "
-            "coord_values={'threshold': 0.03}))[0].data - "
-            "cubes.extract(Constraint(name="
-            "'probability_of_rainfall_rate_above_threshold', "
-            "coord_values={'threshold': 0.03}))[0].data * 0.7) < 0.5"
-        )
-        result = plugin.construct_condition(constraint_list, condition, prob_threshold)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, expected)
-
-
-class Test_format_condition_chain(IrisTest):
-
-    """Test the format_condition_chain method."""
-
-    def test_basic(self):
-        """Test that the format_condition_chain method returns a string."""
-        plugin = WeatherSymbols()
-        conditions = ["condition1", "condition2"]
-        expected = "(condition1) & (condition2)"
-        result = plugin.format_condition_chain(conditions)
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, expected)
-
-    def test_works_with_or(self):
-        """Test that the format_condition_chain method works with OR."""
-        plugin = WeatherSymbols()
-        conditions = ["condition1", "condition2"]
-        expected = "(condition1) | (condition2)"
-        result = plugin.format_condition_chain(conditions, condition_combination="OR")
-        self.assertIsInstance(result, str)
-        self.assertEqual(result, expected)
-
-
-class Test_create_condition_chain(IrisTest):
+class Test_create_condition_chain(Test_WXCode):
     """Test the create_condition_chain method."""
 
     def setUp(self):
         """ Set up queries for testing"""
+        super().setUp()
+        plugin = WeatherSymbols()
```

Comment:
I'm not sure that `plugin = WeatherSymbols()` line is required?

Reply:
You're right, it gets set up in the individual tests so we don't need it here.